src/ui/views/ViewMobileOnboarding.qml

```diff
@@ -23,6 +23,10 @@ VPNFlickable {
     height: parent.height
     interactive: flickContentHeight > height
 
+    VPNMobileStatusBarModifier {
+        statusBarTextColor: VPNMobileStatusBarModifier.StatusBarTextColor.Light
+    }
```

Comment:
On the Jira ticket, it says this page is the only one that has issues with bad contrast and I believed it. I did not manually check all pages.

Reply:
So happy you picked this up! The Jira ticket didn't lie but what if we move this into `main.qml` so that it can be accessed from anywhere once we enable theming/dark mode? Or maybe we don't actually need the `VPNMobileStatusBarModifier` component and we just call `VPN.setStatusBarTextColor(...)` directly as necessary?

Reply:
So, IIUC this component should not be tied to the theme. I wrote it with the purpose to overwrite whatever is the theme default color for pages where a specific status bar text color is needed. That is why I only include it in this specific page, it is always going to have a dark background regardless of theme so it should reliably always have a light status bar text color.

The reason why I added the `VPNMobileStatusBarModifier` abstraction is because it includes the logic to apply the changes once the component is created and reset them once it is destroyed.

Let me know if the reasoning is incorrect :D